src/components/typography/typography-v2.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
```

Comment:
Can be removed and the file type changed to `.ts` instead of `.tsx`. We don't have any JSX in here.

This is what Github is telling me (nice feature):
<img width="652" alt="Screenshot 2021-11-22 at 17 07 02" src="https://user-images.githubusercontent.com/7814926/142895392-872da543-017e-4eec-8838-b83a9537093c.png">

Reply:
Thank you, I've implemented the requested Changes. Please have another look at the PR.